src/Entity/User.php

```diff
@@ -1252,4 +1252,13 @@ public function hasDroit(string $droit): bool
     {
         return $this->getValidUserCentres()->exists(fn (int $index, UserCentre $userCentre) => $userCentre->hasDroit($droit));
     }
+
+    public function formatPhone(): static
+    {
+        if (preg_match('/^0[1-9][0-9]{8}$/', $this->telephone)) {
+            $this->telephone = preg_replace('/^0/', '+33', $this->telephone);
+        }
+
```

Comment:
I wonder if we should check '/^0[6-7][0-9]{8}$/' instead. We should format a phone number with french telephone code if it is a french telephone number